oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/util.js

```diff
@@ -273,3 +274,24 @@ export const goBack = (fallBackURL = "/") => {
     window.location.href = fallBackURL;
   }
 };
+
+// until we start using v4 of react-invenio-forms. They switched to vnd zenodo accept header
+const baseAxiosConfiguration = {
+  withCredentials: true,
+  xsrfCookieName: "csrftoken",
+  xsrfHeaderName: "X-CSRFToken",
+  headers: {
+    Accept: "application/vnd.inveniordm.v1+json",
+    "Content-Type": "application/json",
+  },
+};
+
+export const http = axios.create(baseAxiosConfiguration);
+
+export const encodeUnicodeBase64 = (str) => {
+  return btoa(encodeURIComponent(str));
```

Comment:
SHouldn't be as `window.btoa` ?

Reply:
@mirekys Hey, I think this function is just part of the global namespace, so window. before it is optional. It works like this (without the window.)
![image](https://github.com/user-attachments/assets/e305fff1-3a73-4082-99e0-09ff01bc0f0d)